extensions/Worker.Extensions.Kafka/release_notes.md

```diff
@@ -3,6 +3,10 @@
 - My change description (#PR/#issue)
 -->
 
+### 3.6.0
+- Confluent library upgrade from 1.6.3 to 1.9.0 for fixing duplicate record processing issue (https://github.com/Azure/azure-functions-kafka-extension/pull/357)
+- Fixed the bug for accessing the certificate from relative path instead of absolute path (https://github.com/Azure/azure-functions-kafka-extension/pull/386)
+
 ### 3.5.0
```

Comment:
nit: Remove the 3.5.0 entry here as we already released that. We only need the notes for the next package to be released in the release notes file. Usually we reset the release notes file after we release the packages.

Reply:
Done